target/linux/ramips/patches-6.6/845-pwm-add-mediatek-support.patch

```diff
@@ -172,36 +172,26 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
 +
 +static const struct pwm_ops mtk_pwm_ops = {
 +	.apply = mtk_pwm_apply,
-+	.owner = THIS_MODULE,
```

Comment:
Are you sure this can be removed? The struct is `const` and many other driver still have it in ` struct pwm_ops`.

Reply:
the coccinelle check is titled api/platform_no_drv_owner.cocci and looking at it, it only applies to struct platform_device.

Will add this back.